md/tests/test_api.py

```diff
@@ -0,0 +1,141 @@
+from django.core.urlresolvers import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from md.models import ETHNICITY_CHOICES, PURPOSE_CHOICES
+from md.tests import factories
+
+
+class AgencyTests(APITestCase):
+
+    # Why "aaa"?  It must run first to get the expected agency list.
```

Comment:
I think that if this testcase doesn't run first, there will be more than one agency in the list, and the testcase expects exactly one.  I'll try messing with it to make it more resilient.